src/gitmoji.js

```diff
@@ -168,17 +179,28 @@ class GitmojiCli {
 			},
 			{
 				name: 'reference',
-				message: 'Issue / PR reference #:',
+				message: 'Issue / PR reference:',
 				validate(value) {
 					if (value === '') {
 						return true;
 					}
 					if (value !== null) {
-						const validReference = value.match(/(^[1-9][0-9]*)+$/);
+						let validReference = '';
+						let errorReference = '';
+						switch (config.get('issueFormat')) {
+							case 'jira':
+								validReference = value.match(/^([a-zA-Z]{1,10}-[0-9]+)$/g);
```

Comment:
By default it is three letters and unlimited numbers. However, it is configurable. Based on the [docs here](https://confluence.atlassian.com/adminjiraserver073/changing-the-project-key-format-861253229.html) it suggests `([A-Z][A-Z]+)` which is much looser but I will change it to that.